data/anaconda.conf

```diff
@@ -32,6 +32,9 @@ can_detect_unsupported_hardware = False
 # Should the installer show a warning about removed support for hardware?
 can_detect_support_removed = False
 
+# Should the installer show a warning about enabled SMT?
+can_detect_enabled_smt = False
```

Comment:
I think it should be named ``show_warning_on_enabled_smt`` because the `can_detect...` does not explaining what will happen if we detect that.

Reply:
I am following the pattern of the options above. If we want to use a different pattern, we should rename all these options, but that is for another PR.